script.user.js

```diff
@@ -15,7 +15,7 @@
 // @grant       GM_openInTab
 // @grant       GM_registerMenuCommand
 
-// @version     1.0.14
+// @version     1.0.15
```

Comment:
for noncritical stuff I increase the version separately so no need to change it here

Reply:
Ok 👍